src/app/FakeLib/DotNetCLIHelper.fs

```diff
@@ -1,5 +1,5 @@
 ﻿/// Contains a task which can be used to run dotnet CLI commands.
-module Fake.DotNet
+module Fake.DotNetCli
```

Comment:
sry, didint see this pr in time :cry: 

Np anyway, it's not that bad (i'd like more .net cli than .net sdk).
And maybe is going to change anyway, who knows :smile: